src/snowcli/utils.py

```diff
@@ -582,3 +583,23 @@ def create_project_template(template_name: str):
         f"{os.getcwd()}",
         dirs_exist_ok=True,
     )
+
+
+def get_client_git_version():
+    """
+    Returns the major and minor version of git installed on a user's machine.
+    Args:
+        None
+
+    Returns:
+        str: the major and minor version of git, e.g. "2.25"
+    """
+    output = subprocess.check_output(["git", "--version"], text=True)
+    pattern = r"git version (\d+\.\d+)"
+    match_found = re.search(pattern, output.strip())
+    if match_found:
+        git_version = match_found.group(1)
+        return git_version
+    else:
+        log.error("Could not find a version for git")
+        raise typer.Abort()
```

Comment:
Please use [GitPython](https://gitpython.readthedocs.io/en/stable/tutorial.html). This will make the code more readable andwill help in future.

```py
import git
return git.cmd.Git().version()
```